packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -120,8 +154,8 @@ module.exports.inspect = inspect
  * @return {Promise<Accumulator>}
  */
 function resolveRequest (acc, resolveReducer) {
-  inspect(acc)
-  return rp(acc.options)
+  const _request = inspect(rp, acc)
```

Comment:
anything stopping us from calling this variable `request`? I dont think ive seen any variable on this project with a prefixing `_`.

Reply:
done